tests/shared/utils.spec.js

```diff
@@ -333,4 +333,88 @@ describe('Shared: Utils', function() {
 
   });
 
+  describe('.isValidPercentage()', function() {
+
+    it('should return false if percentage is not a number', function() {
+      m.chai.expect(utils.isValidPercentage('50')).to.be.false;
+    });
+
+    it('should return false if percentage is an integer less than 0', function() {
+      m.chai.expect(utils.isValidPercentage(-1)).to.be.false;
+    });
+
+    it('should return false if percentage is a float less than 0', function() {
+      m.chai.expect(utils.isValidPercentage(-0.1)).to.be.false;
+    });
+
+    it('should return true if percentage is 0', function() {
+      m.chai.expect(utils.isValidPercentage(0)).to.be.true;
+    });
+
+    it('should return true if percentage is an integer greater than 0, but less than 100', function() {
+      m.chai.expect(utils.isValidPercentage(50)).to.be.true;
+    });
+
+    it('should return true if percentage is a float greater than 0, but less than 100', function() {
+      m.chai.expect(utils.isValidPercentage(49.55)).to.be.true;
+    });
+
+    it('should return true if percentage is 100', function() {
+      m.chai.expect(utils.isValidPercentage(100)).to.be.true;
+    });
+
+    it('should return false if percentage is an integer greater than 100', function() {
+      m.chai.expect(utils.isValidPercentage(101)).to.be.false;
+    });
+
+    it('should return false if percentage is a float greater than 100', function() {
+      m.chai.expect(utils.isValidPercentage(100.001)).to.be.false;
+    });
+
+  });
+
+  describe('.percentageToFloat()', function() {
```

Comment:
Maybe `null` would be good to test too considering `null <= 100` is `true`, to make it explicit we avoid this pitfall?